qemu/tests/nvdimm_redis.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+from virttest import error_context
+from virttest import utils_package
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Run nvdimm cases:
+    1) Boot guest with nvdimm device backed by a host file
+    2) Run redis test inside guest
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    vm = env.get_vm(params["main_vm"])
+    session = vm.wait_for_login()
+    error_context.context("Install dependency packages in guest", logging.info)
+    pkgs = params["depends_pkgs"].split()
+    if not utils_package.package_install(pkgs, session):
+        test.cancel("Install dependency packages failed")
+    try:
+        error_context.context("Get redis in guest", logging.info)
+        cmds = []
+        cmds.append(params["get_redis"])
+        cmds.append(params["get_nvml"])
+        cmds.append(params["compile_nvml"])
+        cmds.append(params["compile_redis"])
+        for cmd in cmds:
+            s, o = session.cmd_status_output(cmd, timeout=300)
```

Comment:
Should we increase the time out a bit here ?

Reply:
Did you hit any timeout issue here? The current one works well on x86. Please let me know if you prefer a new one. Thanks.

Reply:
How about set it as 600 ?

Reply:
Sure, updated.